apps/sample-api/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import cors from 'cors'
+import * as dotenv from 'dotenv'
+import express from 'express'
+
+dotenv.config()
+
+const corsOptions = {
+  origin: ['http://localhost:3000', 'http://localhost:5000'],
```

Comment:
Let's make ports configurable? We can probably use [`globalDependencies`](https://turborepo.org/docs/reference/configuration#globaldependencies) in turbo.json or specify path in `dotenv.config`.

Reply:
Updated